tmp/run_sql

```diff
@@ -0,0 +1 @@
+PGPASSWORD=<your_pg_password> psql -f <your_sql_filename>.sql -h pgdb -U postgres postgres
```

Comment:
Similar thing here with the postgres password, just add your pw and your sql file you've been working in.